spec/features/collection/collection_spec.rb

```diff
@@ -287,15 +286,13 @@
   it { expect { visit "/collections/#{collection.id}" }.to_not raise_error }
 
   describe 'paginate collections' do
-    let!(:collection_delete) do
```

Comment:
More bad copy pastey?

Why create a let block here if you are never using the variable? Also this has nothing to do with collection_delete and was copied and pasted here I assume.